<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QDialog" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>461</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Homeworks</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="buttonOpen">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="toolTip">
         <string>open an image file</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images/mainframe.qrc">
          <normaloff>:/open.png</normaloff>:/open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonDeleteContents">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="toolTip">
         <string>close all forms</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images/mainframe.qrc">
          <normaloff>:/1-21.png</normaloff>:/1-21.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="buttonShowList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="toolTip">
         <string>show the list view</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images/mainframe.qrc">
          <normaloff>:/2-3.png</normaloff>:/2-3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>82</width>
          <height>41</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>299</width>
         <height>394</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="NoRole">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="NoRole">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="NoRole">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string> cv 1 </string>
        </attribute>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>83</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Hue</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinHue">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
         <property name="value">
          <number>360</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>83</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sat.</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="spinSaturation">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>80</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonSepiaTone">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>30</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sepia Tone</string>
         </property>
         <property name="iconSize">
          <size>
           <width>41</width>
           <height>41</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonContrast">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>140</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Contrast Transfrom</string>
         </property>
         <property name="iconSize">
          <size>
           <width>41</width>
           <height>41</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinMin">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>190</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinMax">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>190</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>193</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>193</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string> cv 2 </string>
        </attribute>
        <widget class="QPushButton" name="buttonOtsu">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>30</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Otsu Thershold</string>
         </property>
         <property name="iconSize">
          <size>
           <width>41</width>
           <height>41</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonMorp">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>100</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Morphology</string>
         </property>
         <property name="iconSize">
          <size>
           <width>41</width>
           <height>41</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QComboBox" name="cbMorp">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>100</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Dilation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Erosion</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Opening</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Closing</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="buttonLabel">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>150</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Labeling</string>
         </property>
         <property name="iconSize">
          <size>
           <width>41</width>
           <height>41</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string> cv 3 </string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string> cv 4 </string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>394</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="NoBrush">
            <color alpha="128">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="NoBrush">
            <color alpha="128">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="NoBrush">
            <color alpha="128">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/mainframe.qrc"/>
 </resources>
 <connections/>
</ui>
